csharp/src/Drivers/Apache/Hive2/HiveServer2Statement.cs

```diff
@@ -127,6 +130,8 @@ protected async Task ExecuteStatementAsync()
 
         protected internal long BatchSize { get; private set; } = HiveServer2Connection.BatchSizeDefault;
 
+        protected internal int QueryTimeoutSeconds { get; private set; } = HiveServer2Connection.QueryTimeoutDefault;
```

Comment:
... like this is QueryTimeoutSeconds returning "QueryTimeoutDefault"

Reply:
Fixed